applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -323,6 +228,27 @@ static int channel_index_get(const struct bt_conn *conn, uint8_t *index)
 	return -EINVAL;
 }
 
+static int channel_index_free_get(const struct bt_conn *conn, uint8_t *index)
```

Comment:
I think this can be done more elegantly by using some zephyr data structures. Discuss

Reply:
Change to vacant_get